### PR TITLE
Autonav field location

### LINKED ISSUE #8: Use Driver Station for Robot Starting Position
Using FMS data to get which driver station the computer is at to change which path is to be executed

### PR DESCRIPTION
**Pull request by karthikappana35**
Purpose: 
To get autonav starting position from drivers station
Files Changed:
RobotContainer.java - used drivers station class to get a starting position

### CODE REVIEW

**Review by mahisvivemulapalli — APPROVED** [review]
looks good

**Review by PranjalMhetre — APPROVED** [review]

File: src/main/java/frc/robot/RobotContainer.java [hunk 1]
```diff
@@ -121,7 +125,19 @@ private void configureButtonBindings() {
    */
   public Command getAutonomousCommand() {
```
Comment (bjmcternan) [inline src/main/java/frc/robot/RobotContainer.java:126]: I'm not a fan of multiple return statements. It gets hard to follow. Instead, we should have a Command variable which is set on 129, 132, or 135, and then returned on 140

**Review by bjmcternan — CHANGES_REQUESTED** [review]
